spec/migrations/20240502035220_update_n8n_url_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require_relative '../../db/migrate/20240502035220_update_n8n_url'
+
+describe UpdateN8nUrl do
+  # We may want to move this to a support file if this syntax is useful in
+  # other places. Reference:
```

Comment:
I know I've wanted it at times. But as it's not rspec core, I avoided it and just added an expectation in a separate line below.

But I think it's unambiguous and helpful, so I'd support it 👍

Reply:
I think you could do something like : 
```ruby
#...
 }.to change { app.data["destroy"] }.to("https://n8n.openfoodnetwork.org/webhook/remove-enterprise?key=abc&id=123")
  .and change { app.data["link"] }.by(0)
```
But it's not very nice.

Reply:
I tried that. It works with numbers but RSpec tries to find the difference and `string - int` (minus) fails. The minus operator is also not available between two strings.

Reply:
Good to know !